src/App.tsx

```diff
@@ -1,10 +1,27 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import getFoodPantries from 'src/client/getFoodPantries';
+import { FoodPantry } from 'src/utils/types';
+import FoodPantryDisplay from './components/FoodPantryDisplay/index';
 import logo from './logo.svg';
 import './styles/main.css';
 import './App.css';
-import FoodPantryDisplay from './components/FoodPantryDisplay/index';
 
 function App(): JSX.Element {
+  // Pass foodPantries to other components
+  const [foodPantries, setFoodPantries] = useState<FoodPantry[] | undefined>([]);
+
+  useEffect(() => {
+    const gatherData = async () => {
+      try {
+        await setFoodPantries(await getFoodPantries());
+      } catch (e) {
+        console.log(`Error: ${e}`);
+      }
+    };
+
+    gatherData();
+  });
```

Comment:
Needs to have an empty dependency array otherwise this will run on each render instead of just the first
```suggestion
  }, []);
```